src/meta/src/controller/fragment.rs

```diff
@@ -1096,7 +1100,7 @@ impl CatalogController {
         fragment_id: FragmentId,
     ) -> MetaResult<Vec<(ActorId, Vec<ActorId>)>> {
         let inner = self.inner.read().await;
-        let actors: Vec<(ActorId, Vec<ActorId>)> = Actor::find()
+        let actors: Vec<(ActorId, ActorUpstreamActors)> = Actor::find()
```

Comment:
Fixed a bug introduced in #15602 . Cc @xxchan

Reply:
This is so 🤡. I thought it can work as long as it compiles.

Reply:
😄 It is indeed error prone, which can only be discovered at runtime.